```
[2] whatsnew/3.8.rst:59: default role used
[1] whatsnew/3.8.rst:54: trailing whitespace
[1] whatsnew/3.8.rst:56: trailing whitespace
[1] whatsnew/3.8.rst:61: trailing whitespace
3 problems with severity 1 found.
1 problem with severity 2 found.
Makefile:183: recipe for target 'check' failed
```

--- a/Doc/whatsnew/3.8.rst
+++ b/Doc/whatsnew/3.8.rst
@@ -51,14 +51,13 @@ For full details, see the :ref:`changelog <changelog>`.
    Prerelease users should be aware that this document is currently in draft
    form. It will be updated substantially as Python 3.8 moves towards release,
    so it's worth checking back even after reading earlier versions.
-   
+
    Some notable items not yet covered here:
-   
+
    * :pep:`574` - Pickle protocol 5 with out-of-band data buffer support
    * :pep:`578` - Runtime audit hooks for potentially sensitive operations
-   * `python -m asyncio` runs a natively async REPL
+   * ``python -m asyncio`` runs a natively async REPL
    * ...
-   
 
 
 Summary -- Release highlights

